code/datums/diseases/advance/symptoms/flesh_eating.dm

```diff
@@ -56,15 +57,32 @@ Bonus
 			Flesheat(M, A)
 
 /datum/symptom/flesh_eating/proc/Flesheat(mob/living/M, datum/disease/advance/A)
-	var/get_damage = rand(15,25) * power
-	M.take_overall_damage(brute = get_damage, required_status = BODYPART_ORGANIC)
-	if(pain)
-		M.adjustStaminaLoss(get_damage * 2)
-	if(bleed)
-		if(ishuman(M))
-			var/mob/living/carbon/human/H = M
-			H.bleed_rate += 5 * power
-	return 1
+	if(ishuman(M))
```

Comment:
early return this

Reply:
what about the damage dealing part of the proc at the bottom?

Reply:
Run it before